ansible/files/steps/zeppelin/zeppelin.sh

```diff
@@ -17,6 +18,17 @@ nohup Xvfb &
 
 echo 'export DISPLAY=:0.0' | sudo tee -a /etc/zeppelin/conf/zeppelin-env.sh
 
+# Preload Scala packages
+repositories="https://oss.sonatype.org/content/repositories/snapshots"
+packages="com.mozilla.telemetry:moztelemetry_2.11:1.0-SNAPSHOT,vitillo:spark-hyperloglog:1.1.1"
+
+echo ":quit" | spark-shell --master local[1] --repositories $repositories --packages $packages
```

Comment:
This is needed to download all the needed dependencies before a user connects to Zeppelin; without it Zeppelin downloads them when the first Scala cell is executed. The operation can take several minutes which isn't great in terms of UX. Furthermore, it also causes a timeout exception within Zeppelin.